lib/git.sh

```
@@ -1,5 +1,10 @@
 # Git-interaction helpers
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)